pages/file_menu.md

```diff
@@ -10,72 +10,14 @@ edited: 29 November 2015
-<td><b>Ctrl+S</b></td>
-<td>Saves the currently opened file. OCD files will be saved in version 8 format.</td>
-</tr>
-<tr>
-<td> </td>
-<td><b>Save as...</b></td>
-<td><b>Ctrl+Shift+S</b></td>
-<td>Shows the file dialog for choosing a file name and file format to save the currently opened map.</td>
-</tr>
-<tr>
-<td> </td>
-<td><b>Import...</b></td>
-<td> </td>
-<td>Permits the import of maps or data in other formats. Currently the following formats are supported: .omap, .xmap, .ocd (version 6 to 11), .gpx, .osm, .dxf.</td>
-</tr>
-<tr>
-<td> </td>
-<td><b>Export as..</b></td>
-<td> </td>
-<td>Permits the export of the currently opened map to another format. Currently pdf and raster image export are supported.</td>
-</tr>
-<tr>
-<td><img class="small" src="../mapper-images/print.png" width="32" height="32" border="0" alt="" /></td>
-<td><b>Print</b></td>
-<td><b>Ctrl+P</b></td>
-<td><p>Shows the print dialog to print the currently opened map.</td>
-</tr>
-<tr>
-<td> </td>
-<td><b>Settings</b></td>
-<td> </td>
-<td>Opens the <a href="settings.md">settings dialog</a>. (On OS X, the settings are in the usual place instead.)
-</td>
-</tr>
-<tr>
-<td><img class="small" src="../mapper-images/close.png" width="32" height="32" border="0" alt="" /></td>
-<td><b>Close</b></td>
-<td><b>Ctrl+W, Ctrl+F4</b></td>
-<td>Closes the current program window.</td>
-</tr>
-<tr>
-<td> </td>
-<td><b>Exit</b></td>
-<td><b>Ctrl+Q</b></td>
-<td>Exits the program, which will close all open map windows.</td>
-</tr>
-</table>
+|![ ](../mapper-images/new.png)|**New**|**Ctrl+N**|This shows the dialog to create a [new map](new_map.md).|
+|![ ](../mapper-images/open.png)|**Open...**|**Ctrl+O**|Shows the dialog to open an existing map file. Currently supported formats are .omap, .xmap or .ocd (versions 6 to 11).|
+| |**Open recent**| |This shows a list of recently opened files, recognizing that a file to be opened will often have been previously opened in the recent past.|
+|![ ](../mapper-images/save.png)|**Save**|**Ctrl+S**|Saves the currently opened file. OCD files will be saved in version 8 format.|
+| |**Save as...**|**Ctrl+Shift+S**|Shows the file dialog for choosing a file name and file format to save the currently opened map.|
+| |**Import...**| |Permits the import of maps or data in other formats. Currently the following formats are supported: .omap, .xmap, .ocd (version 6 to 11), .gpx, .osm, .dxf.|
+| |**Export as...**| |Permits the export of the currently opened map to another format. Currently pdf and raster image export are supported.|
+|![ ](../mapper-images/print.png)|**Print**|**Ctrl+P**|Shows the print dialog to print the currently opened map.|
+| |**Settings**| |Opens the [settings dialog](settings.md). (On OS X, the settings are in the usual place instead.)|
+|![ ](../mapper-images/close.png)|**Close**|**Ctrl+W, Ctrl+F4**|Closes the current program window.|
+| |**Exit**|**Ctrl+Q**|Exits the program, which will close all open map windows.|

```

Comment:
Fine for now. On the long run, I would prefer to get rid of the table, like we did for some other pages.

Reply:
I see why, although the pages where they have been replaced is a pain too. This is something that has to be tested in Doxygen.